splash-screen/android/src/main/java/com/capacitorjs/plugins/splashscreen/SplashScreen.java

```diff
@@ -45,12 +54,24 @@ public class SplashScreen {
      * @param activity
      */
     public void showOnLaunch(final AppCompatActivity activity) {
-        if (config.getLaunchShowDuration() == 0) {
-            return;
-        }
         SplashScreenSettings settings = new SplashScreenSettings();
         settings.setShowDuration(config.getLaunchShowDuration());
         settings.setAutoHide(config.isLaunchAutoHide());
+
+        // Method can fail if styles are incorrectly set...
+        // If it fails, log error & fallback to old method
+        try {
+            showWithAndroid12API(activity, settings);
+            return;
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
```suggestion
```

Don't print the whole stack trace, it's not very helpful for the users, the warning should be enough.